tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.csproj

```diff
@@ -2,17 +2,22 @@
 
   <PropertyGroup>
     <Version>2.16.0</Version>
-    <Version>$(Version)-beta01</Version> <!-- Unconditionally add a beta suffix to the package, but keep the rest of the name so we can associate it with the rest of the release -->
+    <Version>$(Version)</Version>
```

Comment:
You can delete this line entirely now. There is a version property directly above that is setting the version number

Reply:
Thanks. Done in 05964fbfe0871542759b1d5d2cf1b742f6af58f6